python/langsmith/evaluation/llm_evaluator.py

```diff
@@ -278,15 +332,23 @@ def _prepare_variables(self, run: Run, example: Optional[Example]) -> dict:
 
     def _parse_output(self, output: dict) -> Union[EvaluationResult, EvaluationResults]:
         """Parse the model output into an evaluation result."""
+        run_info = output.get(RUN_KEY)
+        source_run_id = run_info.run_id if run_info is not None else None
         if isinstance(self.score_config, CategoricalScoreConfig):
             value = output["score"]
-            explanation = output.get("explanation", None)
+            explanation = output.get(self.reasoning_key, None)
```

Comment:
I think we should handle if any of (self.reasoning_key, "reasoning", "explanation", "comment") are provided

Reply:
am slightly confused on how something other than self.score_config.reasoning_key would exist. In the `_create_score_json_schema` don't we only set score_config.reasoning_key? I am probably just missing something simple